automata/embedding/data_root_path_types.py

```diff
@@ -0,0 +1,5 @@
+from enum import Enum
+
+
+class DataRootPathType(Enum):
```

Comment:
Hmm, I might have asked for this, but if we are using a configurable env variable to set the data root path then we **should not use an enum**, apologies.